website/content/en/docs/faqs/_index.md

```diff
@@ -109,9 +109,19 @@ by adding an RBAC directive to generate a `config/rbac/role.yaml` with `update`
 
 Now run `make manifests` to update your `role.yaml`.
 
+## Seeing errors about `securityContext.runAsUser: Invalid value: 65532: must be in the ranges: [<NNN>, <MMM>]]`
+	
+For instance, if you are creating the `--kind=memcached` operator with `operator-sdk`. A workaround for this specific instance is to change the generated `config/manager/manager.yaml`:
```

Comment:
```suggestion
Note that, the error can only be checked for some specific cluster's vendors such as OCP. In order to fix the issue,  replace `runAsUser: 65532` with `runAsNonRoot: true` in the `config/manager/manager.yaml` file :
```

Reply:
It is not specific for who are creating an API are creating the `--kind=memcached` operator with `operator-sdk`. Indeed the root cause is because of the vendor. Then, because of this, I do not think that we ought to say that.